docs/feature_led_matrix.md

```diff
@@ -5,6 +5,84 @@ This feature allows you to use LED matrices driven by external drivers. It hooks
 If you want to use RGB LED's you should use the [RGB Matrix Subsystem](feature_rgb_matrix.md) instead.
 
 ## Driver configuration :id=driver-configuration
+---
+### IS31FL3729 :id=is31fl3729
+
+There is basic support for addressable LED matrix lighting with the I2C IS31FL3729 LED controller. To enable it, add this to your `rules.mk`:
+
+```make
+LED_MATRIX_ENABLE = yes
+LED_MATRIX_DRIVER = is31fl3729
+```
+
+You can use between 1 and 4 IS31FL3729 IC's. Do not specify `LED_DRIVER_ADDR_<N>` defines for IC's that are not present on your keyboard. You can define the following items in `config.h`:
+
+| Variable | Description | Default |
+|----------|-------------|---------|
+| `IS31FL3729_I2C_TIMEOUT` | (Optional) How long to wait for i2c messages, in milliseconds | 100 |
+| `IS31FL3729_I2C_PERSISTENCE` | (Optional) Retry failed messages this many times | 0 |
+| `IS31FL3729_CONFIGURATION` | (Optional) Configuration for the Configuration Register | 0x01 |
+| `IS31FL3729_GLOBALCURRENT` | (Optional) Configuration for the Global Current Register | 0x40 |
+| `IS31FL3729_PULLDOWNUP` | (Optional) Configuration for the Pull Up & Pull Down Register | 0x33 |
+| `IS31FL3729_PWM_FREQUENCY` | (Optional) Configuration for the PWM Setting Register | 0x01 |
+| `IS31FL3729_DRIVER_COUNT` | (Required) How many LED driver IC's are present | |
+| `LED_MATRIX_LED_COUNT` | (Required) How many LED lights are present across all drivers | |
+| `LED_DRIVER_ADDR_1` | (Required) Address for the first LED driver | |
+| `LED_DRIVER_ADDR_2` | (Optional) Address for the second LED driver | |
+| `LED_DRIVER_ADDR_3` | (Optional) Address for the third LED driver | |
+| `LED_DRIVER_ADDR_4` | (Optional) Address for the fourth LED driver | |
+
+Drivers does support many matrix layout by using SWS on `IS31FL3729_CONFIGURATION`. Default is using 15 x 9 matrix layout (15 CS pin & 9 SW pin). More info [in datasheet](https://www.lumissil.com/assets/pdf/core/IS31FL3729_DS.pdf)
```

Comment:
```suggestion
IS31FL3729 drivers support multiple matrix layout by using SWS on `IS31FL3729_CONFIGURATION`. Default is using 15 x 9 matrix layout (15 CS pin & 9 SW pin). More info [in datasheet](https://www.lumissil.com/assets/pdf/core/IS31FL3729_DS.pdf)

e.g. `#define IS31FL3729_CONFIGURATION IS31FL3729_CONFIGURATION_SWS_15_9`
```
Maybe need to clarify this a bit more.
Or update the table below this paragraph to have the full define.

Probably should update the naming since this looks redundant naming with `IS31FL3729_CONFIGURATION` included twice.

Reply:
Done, i've just updated the docs for more easier to read.